packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -505,7 +505,7 @@ abstract class IosAssetBundle extends Target {
       targetPlatform: TargetPlatform.ios,
       // Always specify an impeller shader target so that we support runtime toggling and
```

Comment:
I'm guessing we can remove this comment?

Reply:
Done